tests/integration/test_interaction.py

```diff
@@ -695,6 +699,27 @@ def test_per_item_feedback_multiple_misplaced(self):
         for message_element in expected_message_elements:
             self.assertIn(message_element, self._get_popup_content().get_attribute('innerHTML'))
 
+    def test_submit_disabled_during_drop_item(self):
+        def delayed_drop_item(item_attempt, suffix=''):  # pylint: disable=unused-argument
+            # some delay to allow selenium check submit button disabled status while "drop_item"
+            # XHR is still executing
+            time.sleep(0.1)
```

Comment:
Why can't you wait until the button becomes disabled? Sleeps should be avoided at all costs because they are likely to be flaky, and then slow down tests (vs. only waiting until the state changes).